input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -10,6 +10,20 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 
 * ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
 
+* extension contains http://hl7.org/fhir/StructureDefinition/patient-sexParameterForClinicalUse named sexParameterForClinicalUse 0..1 MS
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[code].valueCode = #SHALL:populate-if-known
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-placer"
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][1].extension[code].valueCode = #SHALL:handle
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][1].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-filler"
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[code].valueCode = #SHALL:handle
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-server"
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][3].extension[code].valueCode = #SHALL:able-to-populate
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][3].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-server"
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[code].valueCode = #SHALL:no-error
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-patient"
+* extension[sexParameterForClinicalUse] ^comment = "Requesters can reliably provide this as the sex parameter to be used for the requested service and, when present, as the Placer's requested patient sex parameter for clinical use in the diagnostic request."
```

Comment:
The ballot resolution ticket does not specify that this should be included as a comment on the extension. Was this done in consultation with @michael-wilson-au?

It seems that the comment attempts to clarify the purpose of the extension but it seems unclear and does not provide significant additional value beyond the extension's title and definition: 
- "Requesters can reliably provide this as the sex parameter to be used for the requested service" - what does 'reliable' mean in this context?
- "the sex parameter to be used for the requested service" and "the Placer's requested patient sex parameter" can be interpreted as there could be different values or interpretations for the same concept.

Reply:
The comment was my way to fulfill the (vague) wording in the ticket resolution description. Happy to remove. Or if we want to do it another way, I'll need specific language.

Reply:
Comment still present. 
Implementation guidance wording will need to be worked out with @michael-wilson-au.

Reply:
comment removed.